src/app/shared/documentation-items/documentation-items.ts

```diff
@@ -48,11 +48,8 @@ export const SECTIONS: { [key: string]: DocSection } = {
   },
   [CDK]: {
     name: 'CDK',
-    summary: 'The Component Dev Kit (CDK) is a set of tools that implement common interaction ' +
-      'patterns whilst being unopinionated about their presentation. It represents an ' +
-      'abstraction of the core functionalities found in the Angular Material library, without ' +
-      'any styling specific to Material Design. Think of the CDK as a blank state of well-tested ' +
-      'functionality upon which you can develop your own bespoke components.'
+    summary: 'The Component Dev Kit (CDK) is a set of behavior primitives for building UI' +
```

Comment:
I mean this fixes it for sufficiently large screens... but it's still gonna look weird on smaller ones. I think the right way to do this if you want it centered is to make a wrapper flexbox that centers its item and allow that item to expand as necessary to fill the container, but the text in the item is still aligned left like normal